File: VRSampleApp/VRSampleApp/Classes/PresentationLayer/Modules/VideoPlayerModule/View/ViewController/IMAVideoPlayerViewController.m
//
//  IMAVideoPlayerViewController.m
//  VRSampleApp
//
//  Copyright © 2017 Ooyala Inc. All rights reserved.
//

#import "IMAVideoPlayerViewController.h"
#import <OoyalaIMASDK/OOIMAManager.h>
#import <OoyalaSkinSDK/OoyalaSkinSDK.h>

@interface IMAVideoPlayerViewController ()

@property (nonatomic) OOIMAManager *adsManager;

@end


@implementation IMAVideoPlayerViewController

#pragma mark - Life cycle

- (void)viewDidLoad {
  [super viewDidLoad];
  
  [self configureIMAObjects];
}

#pragma mark - Private functions

- (void)configureIMAObjects {
  // Create ADS manager
  self.adsManager = [[OOIMAManager alloc] initWithOoyalaPlayer:self.skinController.player];
}

@end

File: SwiftVRSkinSampleApp/SwiftVRSkinSampleApp/Classes/PresentationLayer/Modules/VideoPlayerModule/View/ViewController/IMAVideoPlayerViewController.swift
//
//  IMAVideoPlayerViewController.swift
//  VRSkinSampleApp
//
//  Copyright © 2017 Ooyala, Inc. All rights reserved.
//

import UIKit

class IMAVideoPlayerViewController: DefaultVideoPlayerViewController {
  
  // MARK: - Private properties

  private var adsManager: OOIMAManager!
  
  // MARK: - Life cycle
  
  override func viewDidLoad() {
    super.viewDidLoad()
    
    configureObjects()
  }
  
  // MARK: - Private functions
  
  private func configureObjects() {
    
    // Create ADS manager
    adsManager = OOIMAManager(ooyalaPlayer: skinController.player)
  }
  
  
}
